network/src/network.rs

```diff
@@ -429,7 +431,7 @@ impl ServiceHandle for EventHandler {
                 ref address,
                 ref error,
             } => {
-                warn!(target: "network", "DialerError({}) {}", address, error);
+                debug!(target: "network", "DialerError({}) {}", address, error);
```

Comment:
The other errors here also need to be processed, and when they time out, delete their own dail timeout timer.